jfix-zookeeper/src/main/kotlin/ru/fix/zookeeper/discovery/ServiceDiscoveryConfig.kt

```diff
@@ -0,0 +1,7 @@
+package ru.fix.zookeeper.discovery
+
+class ServiceDiscoveryConfig(
+        val rootPath: String,
+        val applicationName: String,
```

Comment:
probably not applicationName but serviceName, there could be several ServiceInstanceIdRegistiry within single application.

Reply:
renamed